workers/unity/Assets/Fps/Scripts/UI/Controllers/Testers/DeploymentListScreenTester.cs

```diff
@@ -0,0 +1,51 @@
+﻿using UnityEngine;
+
+namespace Fps
+{
+    [RequireComponent(typeof(DeploymentListScreenController))]
+    public class DeploymentListScreenTester : MonoBehaviour
```

Comment:
This isn't really testing the screen - its a mock data source.

Reply:
I'd like to know what a proper test class would be for this kind of feature, having never had to write test classes before. I'm going to resolve this as the class should be removed soon.